test/name.cc

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2016-2016, Vasil Dimov
+   Copyright (c) 2019 Linas Vepštas
```

Comment:
Change this to `Copyright (c) 2016-2019, The C++ IPFS client library developers`, I did so in all other files.